### PR TITLE
Add L3 segment

### PR DESCRIPTION
**Pull request by garrettqmartin8**
## Description

Adds ability to parse/generate the `L3` EDI segment.

## Reviewer Notes

Do we need to do any more validation around the parameters?

## Setup

`make server_test`

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-4261) for this change

### CODE REVIEW

**Review by pearl-truss — APPROVED** [review]
Looks good!

File: pkg/edi/segment/l3.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import (
+	"fmt"
+	"strconv"
+)
+
+// L3 represents the L3 EDI segment
+type L3 struct {
+	Weight          float64 `validate:"required_with=WeightQualifier"`
+	WeightQualifier string  `validate:"required_with=Weight"`
+	PriceCents      int
```
Comment (reggieriser) [inline pkg/edi/segment/l3.go:12]: Should we have a `validate` struct tag to make sure we have a non-negative `PriceCents` value?

Comment (reggieriser) [reply pkg/edi/segment/l3.go:12]: Might also want some qualifiers on legal `Weight` and `WeightQualifier` values too.

Comment (garrettqmartin8) [reply pkg/edi/segment/l3.go:12]: @reggieriser I'll address those 👍

Comment (garrettqmartin8) [reply pkg/edi/segment/l3.go:12]: @reggieriser do we actually want to validate that the price isn't negative? @pearl-truss made a good point that fuel surcharge can possibly be negative.

Comment (garrettqmartin8) [reply pkg/edi/segment/l3.go]: @reggieriser updated!

**Pull request merged**